sycl/include/CL/sycl/known_identity.hpp

```diff
@@ -21,8 +21,8 @@ using cl::sycl::detail::is_sgeninteger;
 
 template <typename T, class BinaryOperation>
 using IsReduPlus =
-    bool_constant<std::is_same<BinaryOperation, ONEAPI::plus<T>>::value ||
-                  std::is_same<BinaryOperation, ONEAPI::plus<void>>::value>;
+    bool_constant<std::is_same<BinaryOperation, std::plus<T>>::value ||
```

Comment:
```suggestion
    bool_constant<std::is_same<BinaryOperation, sycl::plus<T>>::value ||
    std::is_same<BinaryOperation, ONEAPI::plus<T>>::value ||
```
I think in terms of code clarity it would be better to list both `sycl::plus` and `ONEAPI::plus` here even though they are both implemented as aliases to the same `std::plus`. The same applies to the other modified lines in this file.

Reply:
Done